cmd/wksctl/profile/disable/disable.go

```diff
@@ -0,0 +1,96 @@
+package disable
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/wksctl/cmd/wksctl/profile/constants"
+	"github.com/weaveworks/wksctl/pkg/git"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "disable",
+	Short: "disable profile",
```

Comment:
please add a `Long` description that explains what this command does to a user who doesn't know it. The explanation could be similar to the description of this PR.

Also applies to `profile enable`.

Reply:
done